### DISCUSSION TITLE
[Resolved] Suddenly tons of 500: INTERNAL_SERVER_ERROR

### DISCUSSION THREAD

**Discussion by simonknittel** [vercel/community#2361] [Help] [answered]
### Summary

I'm suddenly getting tons of the following error on my Vercel-hosted Next.js application.

```
500: INTERNAL_SERVER_ERROR
Code: INTERNAL_FUNCTION_INVOCATION_FAILED
```
![image](https://user-images.githubusercontent.com/8451099/236816557-fed3c96e-eec6-4f08-b580-a543c7e36d59.png)

This error isn't limited to a specific page, but happens on all routes randomly. This concerns all routes like page routes, api routes and even routes like `/api/auth/session` and `/api/auth/_log`

Locally with `npm run dev` and `npm run build && npm start` everything works fine.

I also don't know how to debug this issue since the Vercel Logs don't show any stack trace or similar.
![image](https://user-images.githubusercontent.com/8451099/236815468-b7c74038-26a8-4706-afef-7347c747e86e.png)

I tried rolling back to older commits, but can't find a commit which could be the reason for that.

I'm using Next.js 13.4.1 with the app directory.

**Comment by k9ordon** [comment]
We are experiencing the same issue with all our apps. We didn't have any recent deploys or environment changes, it just happened suddenly. We also tried redeploying with an older version of the Vercel CLI, but that didn't help either.

**Comment by swarnava** [comment]
Hi,

We are investigating this currently https://www.vercel-status.com/incidents/mj596gzq35xw

**Comment by swarnava** [comment, marked as answer]
This is fixed now. https://www.vercel-status.com/incidents/mj596gzq35xw

**Reply by simonknittel** [reply]
Seems to be working, at least for my application. Thank you :)
